src/helpers.js

```diff
@@ -296,7 +296,7 @@ export function rephraseUI(key, original) {
  * @param mimeType
  * @returns {*}
  */
-export function fileDownload({ data, fileName = 'myFile.dat', mimeType = 'application/octet-stream' }) {
+export function fileDownload({ data, _fileName = 'myFile.dat', mimeType = 'application/octet-stream' }) {
```

Comment:
Ah, that's the commented "can't get CSP working" comment. Feels weird to have an unused argument with a default value, but maybe one day we'll figure this out :sweat_smile:

Reply:
Kept this in because of indeed it is used in uncommented code :(